README.md

```diff
@@ -346,21 +353,113 @@ const perfume = new Perfume({
 // Perfume.js: elHeroLogo 1234.00 ms
 ```
 
+### Step Tracking
+
+A Step represents a slice of time in the User Journey where the user is blocked by **system time**. System time is time the system is blocking the user. For example, the time it takes to navigate between screens or fetch critical information from the server. This should not be confused with **cognitive time**, which is the time the user spends thinking about what to do next. User Journey steps should only cover system time.
+
+A Step is defined by an event to start the step, and another event to end the step. These events are referred to as **Marks**.
+
+As an example, a Step could be to navigate from screen A to screen B. The appropriate way to mark a start and end to this step is by marking the start when tapping on the button on screen A that starts the navigation and marking the end when screen B comes into focus with the critical data rendered on the screen.
+
+```typescript
+// Marking the start of the step
+const ScreenA = () => {
+  const handleNavigation = () =>  {
+    ... // Navigation logic
+    // Mark when navigating to screen B
+    markStep('navigate_to_screen_B');
+  }
+
+  ...
+
+  return (
+    <>
+      <Button onPress={handleNavigation} />
+    </>
+  );
+}
+
+// Marking the end of the step
+const ScreenB = () => {
+  const { viewer } = useLazyLoadQuery(
```

Comment:
Make this example simpler, you don't need to use GraphQL. Just a simple fake `fetch` API request will make the point.